src/app/sign-in.component.html

```diff
@@ -0,0 +1,10 @@
+<h2>Sign In</h2>
+Loaded: {{isLoaded}}
```

Comment:
Yes, that works for me too (I can add a button to change the value), but it doesn't give me the behavior I need. Do you see what I'm trying to do? Is there a different technique?